content/en/variables/page.md

```diff
@@ -30,6 +30,15 @@ See [`.Scratch`](/functions/scratch/) for page-scoped, writable variables.
 .AlternativeOutputFormats
 : contains all alternative formats for a given page; this variable is especially useful `link rel` list in your site's `<head>`. (See [Output Formats](/templates/output-formats/).)
 
+.Aliases
+: aliases of this page
+
+.Author
```

Comment:
Drop Author, Authors and IsSection from this list (for different reasons)

Reply:
OK, dropped these commits.